sweep/sweeper.go

```diff
@@ -606,6 +603,12 @@ func (s *UtxoSweeper) collector(blockEpochs <-chan *chainntnfs.BlockEpoch) {
 		return
 	}
 
+	// Create a ticker based on the config duration.
+	ticker := time.NewTicker(s.cfg.TickerDuration)
```

Comment:
+1 for storing the data in the [unified control flow](https://research.swtch.com/pcdata), instead of instances of independent control flow modifying the same piece of data

Reply:
oh nice blog post!